lib/cli.rb

```diff
@@ -55,8 +57,8 @@ class Cli
 
   GLI::Commands::Help.skips_post = false
 
-  def self.validate_number_of_arguments(defined, parsed)
-    if defined.any?(&:multiple?) && parsed.empty?
+  def self.validate_command_line(defined, parsed)
```

Comment:
Assignment Branch Condition size for validate_command_line is too high. [20.32/15]